### PR TITLE
Match chinese language codes to languages

### PR DESCRIPTION
**Pull request by franbull**
Actually maybe we can swap out jed/locale for use of https://github.com/jquery/globalize ? I'll look at that in the morning.

### CODE REVIEW

**Comment by franbull** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;xhw6TLcT&#x2F;4925-i18n-chrome-isn-t-sending-zh-hant-it-s-sending-zh-tw"> I18n: Chrome isn&#39;t sending &#39;zh-Hant&#39; it&#39;s sending &#39;zh-TW&#39;</a>

**Comment by franbull** [comment]
No we can't https://github.com/jquery/globalize/blob/master/doc/api/core/locale.md has

> LanguageMatching TBD (CLDR's spec http://www.unicode.org/reports/tr35/#LanguageMatching).

**Comment by franbull** [comment]
So I think that either we port https://godoc.org/golang.org/x/text/language (which I'd like to do but I would expect it to take a couple of days) or go with this short term band-aid.

**Comment by franbull** [comment]
Actually, as the minutes pass I'm getting less and less comfortable with this fix. I'm going to take a swing at porting the golang thing.

**Comment by franbull** [comment]
Resubmitting for consideration as most expedient.

File: src/index.coffee [hunk 1]
```diff
@@ -12,6 +12,10 @@ app = (supported) ->
     do next
 
 class app.Locale
+  @substitutions =
```
Comment (brettkiefer) [inline src/index.coffee:25]: A comment here is probably a good thing. If not, at least in the commit message, but this is funky enough for one here.

**Comment by brettkiefer** [comment]
👍

**Pull request merged**